src/components/Marketplace/FarmCard.js

```diff
@@ -216,32 +238,34 @@ export default function FarmCard(props) {
               />
             ))}
           </div>
-          { /* Show farming practice tags if expanded */
-            isExpanded
-            && farmingPracticeTags.length >= 1 && (
-              <Grid
-                container
-                direction="row"
-                justify="flex-end"
-                alignItems="center"
-                className={classes.farmingTagsContainer}
-              >
-                <Typography
-                  className={classes.farmingText}
-                >
-                  Farming Practice:
-                </Typography>
-                {farmingPracticeTags.map((tag) => (
-                  <Chip
-                    label={tag}
-                    className={classes.farmingTags}
-                    size="small"
-                    variant="default"
-                  />
-                ))}
-              </Grid>
-            )
-          }
+          {/* // */}
+          { isExpanded
+         && (
+         <Grid
+           container
+           direction="row"
+           justifyContent="flex-start"
+           alignItems="center"
+           className={classes.farmingTagsContainer}
+         >
+           <Typography
+             className={classes.farmingText}
+           >
+             Farming Practice:
+           </Typography>
+           {farmingPracticeTags.map((tag) => (
+             <Chip
+               label={tag}
+               className={classes.farmingTags}
+               size="small"
+               variant="default"
+             />
+           ))}
+         </Grid>
+         )}
+          {/* // */}
+
+          {/* original farm tag content here */}
```

Comment:
Remove this comment if it's not needed, or if it's useful could you edit it? Not sure what it means